sdk/python/feast/feature_store.py

```diff
@@ -1296,32 +1322,31 @@ def get_online_features(
                     str(self.repo_path.absolute()), self.config, self
                 )
 
+            entity_native_values: Dict[str, List[Any]]
```

Comment:
We have both entity_native_values and native_entity_values and that's hella confusing